src/api/mutations/useLogout.ts

```diff
@@ -0,0 +1,9 @@
+import { useMutation } from '@tanstack/react-query';
+
+import { fetcher } from '@/api';
+
+const postLogout = async (): Promise<void> => {
+  await fetcher.post<void>('auth/logout', { json: {} });
```

Comment:
```suggestion
  await fetcher.post<void>('auth/logout');
```

`json: {}` 부분이 꼭 필요하지 않다면 지워주세요.